internal/bufferedwrites/upload_handler.go

```diff
@@ -171,6 +171,29 @@ func (uh *UploadHandler) Finalize() (*gcs.MinObject, error) {
 	return obj, nil
 }
 
+// Flush uploads any data in the write buffer.
+func (uh *UploadHandler) Flush() (int64, error) {
```

Comment:
Flush in fs.go has a different meaning. Can you rename this also to FlushPendingWrites